src/sidebar/services/test/groups-test.js

```diff
@@ -176,6 +177,26 @@ describe('groups', function() {
       });
     });
 
+    it('combines groups from all 3 endpoints if community-groups feature flag is set and there is a selectedGroup', () => {
+      const svc = service();
+
+      fakeSettings.group = 'selected-id';
+      const groups = [
+        { id: 'groupa', name: 'GroupA' },
+        { id: 'groupb', name: 'GroupB' },
+        { id: fakeSettings.group, name: 'Selected Group' },
+      ];
+
+      fakeApi.profile.groups.read.returns(Promise.resolve(groups));
```

Comment:
Since this call returns all three groups here, it looks like this test would pass even if the selected group was not set.